common/app/conf/GoogleAuth.scala

```diff
@@ -0,0 +1,52 @@
+package conf
+
+import conf.Configuration.OAuthCredentials
+import com.amazonaws.auth.AWSCredentialsProviderChain
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.simplesystemsmanagement.AWSSimpleSystemsManagementClientBuilder
+import com.gu.googleauth.{AntiForgeryChecker, GoogleAuthConfig}
+import com.gu.play.secretrotation.{SnapshotProvider, TransitionTiming}
+import play.api.http.HttpConfiguration
+
+import java.time.Duration.{ofHours, ofMinutes}
+
+case class GoogleAuth(
```

Comment:
Putting this in `common` brings us back to an interesting debate. It's "simple" and easy to put absolutely anything you need to share into `common`, but as it turns out that is quite a lot.  That means we bundle loads of unnecessary code in some apps, and `common` itself is generally hard to reason about because it has so many responsibilities and so much code.

The (or at least one?) alternative of lots of focussed micro-projects sounds like a quite complex situation too. For example we once had a `standalone` app that was used by 2 preview-like apps, which we removed for the sake of simplicity 😆 

I don't mean to suggest this is something to address in this PR, but I thought it was something you might want to consider for the future (even if after consideration you do nothing.)